build-tools/packages/version-tools/api-report/version-tools.api.md

```diff
@@ -25,7 +25,11 @@ export function detectBumpType(v1: semver.SemVer | string | null, v2: semver.Sem
 export function detectVersionScheme(rangeOrVersion: string | semver.SemVer): VersionScheme;
 
 // @public
-export function fromInternalScheme(internalVersion: semver.SemVer | string, allowPrereleases?: boolean, allowAnyPrereleaseId?: boolean): [publicVersion: semver.SemVer, internalVersion: semver.SemVer, prereleaseIndentifier: string];
+export function fromInternalScheme(internalVersion: semver.SemVer | string, allowPrereleases?: boolean, allowAnyPrereleaseId?: boolean): [
```

Comment:
We're formatting the api report files? It feels weird... Or is this because some task order change?

Reply:
I think we can disable this. Looking into it.

Reply:
api-extractor itself seems to be making this change. No idea why.

Reply:
There are cases where code snippets preserve the formatting from the source-code. I would consider it an API-Extractor bug (albeit a pretty minor one). But I have definitely seen this behavior before in response to formatting changes in the code.